### ISSUE TITLE
Python 3.x Compatibility

### ISSUE THREAD

**Issue by rlkiser** [deltaray/cloudperm#5]
I realize it's probably a long road, but it would be good to get to Python 3 if possible before too many dependencies shack up and make migration harder.

**Comment by deltaray** [comment]
Agreed. I've already done some of the work such as switching prints to using print() and a few other things.

**Comment by rlkiser** [comment]
If you can put together a list of issues I can start work on them.
